### PR TITLE
Ra 1845:Remove covid-19 concepts from reference Application Metadata

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RA-1845

Removing covid-19  concepts in reference application, since during last release, this work was added by different tickets hence its still work in progress
cc @dkayiwa  @ibacher  @mozzy11

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java [hunk 1]
```diff
@@ -93,16 +93,13 @@ private void installConcepts() {
         	Context.flushSession(); //Flush so that purges are not deferred until after data import
         	
         	DataImporter dataImporter = Context.getRegisteredComponent("dataImporter", DataImporter.class);
-            dataImporter.importData("Reference_Application_Concepts-25.xml");
-            dataImporter.importData("Covid-19_Concepts-1.xml");
+            dataImporter.importData("Reference_Application_Concepts-24.xml");
```
Comment (mozzy11) [inline api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java:96]: did you just rename this `Reference_Application_Concepts-25.xml`   to `Reference_Application_Concepts-24.xml`   ??
are you sure we have a package named `Reference_Application_Concepts-24.xml`  ??

File: api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java [hunk 2]
```diff
@@ -93,16 +93,13 @@ private void installConcepts() {
...
             
             if (ModuleUtil.compareVersion(OpenmrsConstants.OPENMRS_VERSION_SHORT, "2.2") < 0) {
-                dataImporter.importData("Reference_Application_Numeric_Concepts-25-pre2.x.xml");
-                dataImporter.importData("Covid-19_Numeric_Concepts-1_pre2.x.xml");
+                dataImporter.importData("Reference_Application_Numeric_Concepts-24-pre2.x.xml");
 			}
             else {
-                dataImporter.importData("Reference_Application_Numeric_Concepts-25-2.x.xml");
-                dataImporter.importData("Covid-19_Numeric_Concepts-1_2.x.xml");
+                dataImporter.importData("Reference_Application_Numeric_Concepts-24-2.x.xml");
```
Comment (mozzy11) [inline api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java:102]: same here , do we have any package named `Reference_Application_Numeric_Concepts-24-2.x.xml`  ??

Comment (mozzy11) [reply api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java:102]: check the resources folder for the right package names to import   https://github.com/openmrs/openmrs-module-referencemetadata/tree/master/api/src/main/resources

File: api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java (hunk 1 above)
Comment (sherrif10) [reply api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java]: Sorry for that mistake, i have resolved

File: api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java (hunk 2 above)
Comment (sherrif10) [reply api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java]: @mozzy11 i have resolved that thanks

**Pull request merged**